lib/UnoCore/Targets/Android/com/fuse/App.java

```diff
@@ -208,6 +208,7 @@ public void onActivityResult (int arg0, int arg1, android.content.Intent arg2)
     //------------------------------------------------------------
 
     private String pendingURI = "";
+    private String[] associatedDomains;
```

Comment:
I don't think you need a member-variable for this, a local variable should do just fine, no?

Reply:
I guess so, just thought if someone else needed it in the future they could use it but sure, lemme localise it

Reply:
Ok, sorted ;)